apps/andi/README.md

```diff
@@ -18,8 +18,7 @@ Interactions with Redis are abstracted with `smartcitiesdata.smart_city*` functi
 - Install dependencies with `mix deps.get`
 - `cd assets` and `npm i`
 - `MIX_ENV=integration mix docker.start`
-- `MIX_ENV=integration mix ecto.create && MIX_ENV=integration mix ecto.migrate`
-- Start Phoenix endpoint locally with `AUTH0_CLIENT_SECRET="<auth_client_secret>" MIX_ENV=integration iex -S mix phx.server`
+- Start Phoenix endpoint locally with `AUTH0_CLIENT_SECRET="<auth_client_secret>" MIX_ENV=integration iex -S mix start`
```

Comment:
the reason I changed this was because this command doesn't start andi interactively and is less clear about the ecto setup. I feel like it would be helpful to give people more info but I might be wrong

Reply:
I don't really have a preference on this. I can change it back